validation/exclusions_test.go

```diff
@@ -0,0 +1,68 @@
+package validation
+
+import (
+	"regexp"
+	"testing"
+)
+
+func skipIfExcluded(t *testing.T, chainID uint64) {
+	for pattern := range exclusions {
+		matches, err := regexp.Match(pattern, []byte(t.Name()))
+		if err != nil {
+			panic(err)
+		}
+		if matches && exclusions[pattern][chainID] {
+			t.Skip("Excluded!")
+		}
+	}
+}
+
+var exclusions = map[string]map[uint64]bool{
+	"ChainID_RPC_Check": {
+		11155421: true, // sepolia-dev-0/oplabs-devnet-0   No Public RPC declared
+		11763072: true, // sepolia-dev-0/base-devnet-0     No Public RPC declared
+	},
+	"GPO_Params": {
+		11155421: true, // sepolia-dev-0/oplabs-devnet-0   (no public endpoint)
+		11763072: true, // sepolia-dev-0/base-devnet-0     (no public endpoint)
+	},
+	"L2OO_Params": {
+		10:        true, // OP mainnet - Upgraded to fault proofs
+		999999999: true, // sepolia/zora  Incorrect finalizationPeriodSeconds, 604800 is not within bounds [12 12]
+		1740:      true, // sepolia/metal Incorrect finalizationPeriodSeconds, 604800 is not within bounds [12 12]
+		919:       true, // sepolia/mode  Incorrect finalizationPeriodSeconds, 180 is not within bounds [12 12]
+		11155420:  true, // sepolia/op No L2OO because this chain uses Fault Proofs https://github.com/ethereum-optimism/superchain-registry/issues/219
+		11155421:  true, // oplabs-sepolia-devnet-0 No L2OO because this chain uses Fault Proofs https://github.com/ethereum-optimism/superchain-registry/issues/219
```

Comment:
Now these exclusions are in one place, it is easier to see how we can have fewer of them. If the standard config requires at least fault proofs, we ought to be able to drop this test entirely (and a bunch of other code besides).